File: pset1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float x;
    do
    {
        x = get_float("How much changed is owned: \n");
    }
    while (x <= 0);
    int coins = round(x * 100);
    int result = 0;
    do
    {
        if(coins >= 25)
        {
            result += 1;
            coins -= 25;
        }
        else if(coins >= 10)
        {
            result += 1;
            coins -= 10;
        }
        else if(coins >= 5)
        {
            result += 1;
            coins -= 5;
        }
        else if(coins >= 1)
        {
            result += 1;
            coins -= 1;
        }
    }
    while (coins > 0);
    printf("%i\n", result);
}

File: pset6/cash/cash.py
from cs50 import get_float

x = 0
while x <= 0:
    x = get_float("How much changed is owned: \n")

x = int(round(x * 100))
coins = 0

while x > 0:
    if x >= 25:
        coins += 1
        x -= 25
    elif x >= 10:
        coins += 1
        x -= 10
    elif x >= 5:
        coins += 1
        x -= 5
    elif x >= 1:
        coins += 1
        x -= 1
print(coins)        